### PR TITLE
feat(theme): don't print navbar when disabled in frontmatter (narrow screens)

### LINKED ISSUE #3086: Navbar visible on print when navbar param equals false
### Describe the bug

When I hide the navbar (parameter "navbar: false") it works correctly on normal browser navigation, but when I print a page something is wrong.

On the printed page the navbar is not hide and is shown with the text "Return to top" inside.

### Reproduction

Create a new vitepress project.

In the index.md file, write the following code:
```md
---
navbar: false
---

# Hello
```

Run vitepress dev, brose to the site, call print (Ctrl + P) and print the page or save as pdf.

### Expected behavior

The navbar must be hide also on pdf.

### System Info

```Text
System:
    OS: macOS 12.6.9
    CPU: (4) x64 Intel(R) Core(TM) i5-5250U CPU @ 1.60GHz
    Memory: 34.23 MB / 8.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 18.18.1 - ~/.nvm/versions/node/v18.18.1/bin/node
    npm: 9.8.1 - ~/.nvm/versions/node/v18.18.1/bin/npm
  Browsers:
    Chrome: 118.0.5993.70
    Safari: 16.6
  npmPackages:
    vitepress: ^1.0.0-rc.21 => 1.0.0-rc.21
```

### Additional context

_No response_

### Validations

- [X] Check if you're on the [latest VitePress version](https://github.com/vuejs/vitepress/releases/latest).
- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.dev).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

### PR DESCRIPTION
**Pull request by userquin**
This PR includes a global style for `screen-only`: we cannot hide `VPLocalNav`  on narrow screens (we lost the navigation), `screen-only` class added when disabled in frontmatter `navbar: false`.

**missing navbar**
imagen

**navbar: false**
imagen

closes #3086

### CODE REVIEW

**Comment by userquin** [comment]
I need to test this PR on large screen, on my laptop the print preview using narrow screen: maybe we need to also include the logic in the `VPNavBar`

File: src/client/theme-default/styles/base.css [hunk 1]
```diff
@@ -250,3 +250,9 @@ mjx-container {
 mjx-container > svg {
   margin: auto;
 }
+
+@media print {
+  .screen-only {
```
Comment (brc-dd) [inline src/client/theme-default/styles/base.css:255]: maybe move this to src/client/theme-default/styles/utils.css ?

Comment (userquin) [reply src/client/theme-default/styles/base.css]: done

**Comment by userquin** [comment]
@brc-dd I can prepare a new PR to include print options, it will go to the branch of this PR (title will be `feat(theme): add print options` and will respect fm sidebar and navbar options when disabled), what do you think?

**Pull request closed without merging**